// @vuepress/core/.temp/dynamic/vuepress_blog/paginations.js
/**
 * Generated by "@vuepress/plugin-blog"
 */

import sorters from './pageSorters'
import filters from './pageFilters'

export default [{
  pid: "tags",
  id: "Algorithm 算法",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Algorithm 算法/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "Discrete Mathematics 离散数学",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Discrete Mathematics 离散数学/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "Calculus 微积分",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Calculus 微积分/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "Mathematics 数学",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Mathematics 数学/","interval":[0,3]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "SeqList 顺序表",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/SeqList 顺序表/","interval":[0,5]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "BinaryTree 二叉树",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/BinaryTree 二叉树/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "Tree 树",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Tree 树/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "Graph 图",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Graph 图/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "LinkList 链表",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/LinkList 链表/","interval":[0,5]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "Stack 栈",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Stack 栈/","interval":[0,5]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "Algorithm 算法",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/Algorithm 算法/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "Calculus 微积分",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/Calculus 微积分/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "Probability Theory 概率论",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/Probability Theory 概率论/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "Physic 物理",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/Physic 物理/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "深大数据结构OJ",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/深大数据结构OJ/","interval":[0,9]},{"path":"/categories/深大数据结构OJ/page/2/","interval":[10,15]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "DataStructer 数据结构",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/DataStructer 数据结构/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
}]
