// @vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "F:\\WebstormProjects\\inverseda_blog\\node_modules\\@vuepress\\core\\lib\\client\\components\\GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-36575d76",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-36575d76").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-241c36e8",
    path: "/blogs/algorithm/2022/How%20to%20calculate%20the%20time%20complexity.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-241c36e8").then(next)
    },
  },
  {
    path: "/blogs/algorithm/2022/How to calculate the time complexity.html",
    redirect: "/blogs/algorithm/2022/How%20to%20calculate%20the%20time%20complexity.html"
  },
  {
    path: "/blogs/algorithm/2022/How to calculate the time complexity.html",
    redirect: "/blogs/algorithm/2022/How%20to%20calculate%20the%20time%20complexity.html"
  },
  {
    name: "v-e3dfdf2e",
    path: "/blogs/Calculus/2022/Jaccobi.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-e3dfdf2e").then(next)
    },
  },
  {
    name: "v-49a15c7b",
    path: "/blogs/probability%20theory/2022/test%20for%20high%20school.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-49a15c7b").then(next)
    },
  },
  {
    path: "/blogs/probability theory/2022/test for high school.html",
    redirect: "/blogs/probability%20theory/2022/test%20for%20high%20school.html"
  },
  {
    path: "/blogs/probability theory/2022/test for high school.html",
    redirect: "/blogs/probability%20theory/2022/test%20for%20high%20school.html"
  },
  {
    name: "v-868e9f7a",
    path: "/blogs/physic/2022/electric%20and%20magnet.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-868e9f7a").then(next)
    },
  },
  {
    path: "/blogs/physic/2022/electric and magnet.html",
    redirect: "/blogs/physic/2022/electric%20and%20magnet.html"
  },
  {
    path: "/blogs/physic/2022/electric and magnet.html",
    redirect: "/blogs/physic/2022/electric%20and%20magnet.html"
  },
  {
    name: "v-008222b0",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab1_SeqList/b.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-008222b0").then(next)
    },
  },
  {
    name: "v-4f7c0add",
    path: "/blogs/dataStructer/2022/Tree%20And%20Graph.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4f7c0add").then(next)
    },
  },
  {
    path: "/blogs/dataStructer/2022/Tree And Graph.html",
    redirect: "/blogs/dataStructer/2022/Tree%20And%20Graph.html"
  },
  {
    path: "/blogs/dataStructer/2022/Tree And Graph.html",
    redirect: "/blogs/dataStructer/2022/Tree%20And%20Graph.html"
  },
  {
    name: "v-71e032f2",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab1_SeqList/a.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-71e032f2").then(next)
    },
  },
  {
    name: "v-e1b7db24",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab1_SeqList/c.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-e1b7db24").then(next)
    },
  },
  {
    name: "v-1dc6022c",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab1_SeqList/d.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1dc6022c").then(next)
    },
  },
  {
    name: "v-f860ccbc",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab2_LinkedList/b.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f860ccbc").then(next)
    },
  },
  {
    name: "v-70b5befc",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab2_LinkedList/a.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-70b5befc").then(next)
    },
  },
  {
    name: "v-a7301c2c",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab1_SeqList/e.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-a7301c2c").then(next)
    },
  },
  {
    name: "v-07b6e83c",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab2_LinkedList/d.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-07b6e83c").then(next)
    },
  },
  {
    name: "v-3ffa12c2",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab2_LinkedList/c.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3ffa12c2").then(next)
    },
  },
  {
    name: "v-8f61f5fc",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab2_LinkedList/e.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-8f61f5fc").then(next)
    },
  },
  {
    name: "v-670eaa18",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab3_Stack/a.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-670eaa18").then(next)
    },
  },
  {
    name: "v-69f9d596",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab3_Stack/b.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-69f9d596").then(next)
    },
  },
  {
    name: "v-6ce50114",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab3_Stack/c.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6ce50114").then(next)
    },
  },
  {
    name: "v-6fd02c92",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab3_Stack/d.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6fd02c92").then(next)
    },
  },
  {
    name: "v-72bb5810",
    path: "/blogs/szuDataStructerOJ/2021_2022/Lab3_Stack/e.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-72bb5810").then(next)
    },
  },
  {
    name: "v-b1564aac",
    path: "/tag/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tags", "v-b1564aac").then(next)
    },
    meta: {"pid":"tags","id":"tags"}
  },
  {
    path: "/tag/index.html",
    redirect: "/tag/"
  },
  {
    name: "v-ef9325c4",
    path: "/categories/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("FrontmatterKey", "v-ef9325c4").then(next)
    },
    meta: {"pid":"categories","id":"categories"}
  },
  {
    path: "/categories/index.html",
    redirect: "/categories/"
  },
  {
    name: "v-6319eb4e",
    path: "/timeline/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("TimeLines", "v-6319eb4e").then(next)
    },
    meta: {"pid":"timeline","id":"timeline"}
  },
  {
    path: "/timeline/index.html",
    redirect: "/timeline/"
  },
  {
    name: "v-4a8d3abe",
    path: "/tag/Algorithm 算法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-4a8d3abe").then(next)
    },
    meta: {"pid":"tags","id":"Algorithm 算法"}
  },
  {
    path: "/tag/Algorithm 算法/index.html",
    redirect: "/tag/Algorithm 算法/"
  },
  {
    name: "v-a63d8fce",
    path: "/tag/Discrete Mathematics 离散数学/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-a63d8fce").then(next)
    },
    meta: {"pid":"tags","id":"Discrete Mathematics 离散数学"}
  },
  {
    path: "/tag/Discrete Mathematics 离散数学/index.html",
    redirect: "/tag/Discrete Mathematics 离散数学/"
  },
  {
    name: "v-a472b8ca",
    path: "/tag/Calculus 微积分/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-a472b8ca").then(next)
    },
    meta: {"pid":"tags","id":"Calculus 微积分"}
  },
  {
    path: "/tag/Calculus 微积分/index.html",
    redirect: "/tag/Calculus 微积分/"
  },
  {
    name: "v-7af59299",
    path: "/tag/Mathematics 数学/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-7af59299").then(next)
    },
    meta: {"pid":"tags","id":"Mathematics 数学"}
  },
  {
    path: "/tag/Mathematics 数学/index.html",
    redirect: "/tag/Mathematics 数学/"
  },
  {
    name: "v-22c8e09c",
    path: "/tag/SeqList 顺序表/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-22c8e09c").then(next)
    },
    meta: {"pid":"tags","id":"SeqList 顺序表"}
  },
  {
    path: "/tag/SeqList 顺序表/index.html",
    redirect: "/tag/SeqList 顺序表/"
  },
  {
    name: "v-4aa1ce92",
    path: "/tag/BinaryTree 二叉树/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-4aa1ce92").then(next)
    },
    meta: {"pid":"tags","id":"BinaryTree 二叉树"}
  },
  {
    path: "/tag/BinaryTree 二叉树/index.html",
    redirect: "/tag/BinaryTree 二叉树/"
  },
  {
    name: "v-595060ea",
    path: "/tag/Tree 树/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-595060ea").then(next)
    },
    meta: {"pid":"tags","id":"Tree 树"}
  },
  {
    path: "/tag/Tree 树/index.html",
    redirect: "/tag/Tree 树/"
  },
  {
    name: "v-549f4b1f",
    path: "/tag/Graph 图/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-549f4b1f").then(next)
    },
    meta: {"pid":"tags","id":"Graph 图"}
  },
  {
    path: "/tag/Graph 图/index.html",
    redirect: "/tag/Graph 图/"
  },
  {
    name: "v-a879729e",
    path: "/tag/LinkList 链表/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-a879729e").then(next)
    },
    meta: {"pid":"tags","id":"LinkList 链表"}
  },
  {
    path: "/tag/LinkList 链表/index.html",
    redirect: "/tag/LinkList 链表/"
  },
  {
    name: "v-926e8866",
    path: "/tag/Stack 栈/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Tag", "v-926e8866").then(next)
    },
    meta: {"pid":"tags","id":"Stack 栈"}
  },
  {
    path: "/tag/Stack 栈/index.html",
    redirect: "/tag/Stack 栈/"
  },
  {
    name: "v-580281ad",
    path: "/categories/Algorithm 算法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-580281ad").then(next)
    },
    meta: {"pid":"categories","id":"Algorithm 算法"}
  },
  {
    path: "/categories/Algorithm 算法/index.html",
    redirect: "/categories/Algorithm 算法/"
  },
  {
    name: "v-0eb5f6a7",
    path: "/categories/Calculus 微积分/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-0eb5f6a7").then(next)
    },
    meta: {"pid":"categories","id":"Calculus 微积分"}
  },
  {
    path: "/categories/Calculus 微积分/index.html",
    redirect: "/categories/Calculus 微积分/"
  },
  {
    name: "v-a2e9ba88",
    path: "/categories/Probability Theory 概率论/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-a2e9ba88").then(next)
    },
    meta: {"pid":"categories","id":"Probability Theory 概率论"}
  },
  {
    path: "/categories/Probability Theory 概率论/index.html",
    redirect: "/categories/Probability Theory 概率论/"
  },
  {
    name: "v-e113af74",
    path: "/categories/Physic 物理/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-e113af74").then(next)
    },
    meta: {"pid":"categories","id":"Physic 物理"}
  },
  {
    path: "/categories/Physic 物理/index.html",
    redirect: "/categories/Physic 物理/"
  },
  {
    name: "v-67dc5293",
    path: "/categories/深大数据结构OJ/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-67dc5293").then(next)
    },
    meta: {"pid":"categories","id":"深大数据结构OJ"}
  },
  {
    path: "/categories/深大数据结构OJ/index.html",
    redirect: "/categories/深大数据结构OJ/"
  },
  {
    name: "v-5d4300bb",
    path: "/categories/DataStructer 数据结构/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Category", "v-5d4300bb").then(next)
    },
    meta: {"pid":"categories","id":"DataStructer 数据结构"}
  },
  {
    path: "/categories/DataStructer 数据结构/index.html",
    redirect: "/categories/DataStructer 数据结构/"
  },
  {
    name: "v-412b706c",
    path: "/categories/深大数据结构OJ/page/2/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-412b706c").then(next)
    },
    meta: {"pid":"categories","id":"深大数据结构OJ"}
  },
  {
    path: "/categories/深大数据结构OJ/page/2/index.html",
    redirect: "/categories/深大数据结构OJ/page/2/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
